tests/test_validators.py

```diff
@@ -1,4 +1,5 @@
 import pytest
+import referencing
```

Comment:
Is this available from the `jsonschema` dependency?

Reply:
It is -- https://python-jsonschema.readthedocs.io/en/latest/referencing/